game/Doom/Game/p_switch.cpp

```diff
@@ -379,6 +381,11 @@ void P_ChangeSwitchTexture(line_t& line, const bool bUseAgain) noexcept {
 // Assumes the line has a special and only attempts to activate the front side of the line, since that is all that is allowed.
 //------------------------------------------------------------------------------------------------------------------------------------------
 bool P_UseSpecialLine(mobj_t& mobj, line_t& line) noexcept {
+    // PsyDoom: disable exits for deathmatch, if set.
+    #if PSYDOOM_MODS
+        bool exitDisabled = (gNetGame == gt_deathmatch && Game::gSettings.bExitDisabled && Game::gSettings.fragLimit > 0);
+    #endif
```

Comment:
Minor: mark this value `const` since it does not change below. This makes the code easier to read since you can immediately discount the possibility of the value changing elsewhere.

Reply:
79d875bac001d3bb14e7a9d2a82b2ffb836a3c32